node.go

```diff
@@ -129,6 +132,78 @@ func New(value interface{}) *Node {
 	return rootNode
 }
 
+// NewFromFile returns a new root node by parsing file using specified read parser.
+func NewFromFile(filename, parser string) (*Node, error) {
+	readParser, err := storage.NewReadParserFromString(parser)
+	if err != nil {
+		return nil, err
+	}
+
+	data, err := storage.LoadFromFile(filename, readParser)
+	if err != nil {
+		return nil, err
+	}
+
+	return New(data), nil
+}
+
+// NewFromReader returns a new root node by parsing from Reader using specified read parser.
+func NewFromReader(reader io.Reader, parser string) (*Node, error) {
+	readParser, err := storage.NewReadParserFromString(parser)
+	if err != nil {
+		return nil, err
+	}
+
+	data, err := storage.Load(readParser, reader)
+	if err != nil {
+		return nil, err
+	}
+
+	return New(data), nil
+}
+
+// Write writes data to file using specified write parser and options.
+func (n *Node) WriteToFile(filename, parser string, compact, escapeHTML bool) error {
```

Comment:
The prevent an infinitely growing arguments list here, perhaps it would be better to accept a list of storage.ReadWriteOption instead